editor.planx.uk/src/ui/shared/Input.tsx

```diff
@@ -32,6 +38,7 @@ export interface Props extends InputBaseProps {
   large?: boolean;
   bordered?: boolean;
   errorMessage?: string;
+  textLength?: string;
```

Comment:
Added this prop so you can set character limit based on `type`

Reply:
I still need something to create logic on excluding `"Short"` text input types from the character count logic, this is something I would need from the props when specifying the "type"? 

`props.type` in this instance will only be "text" for the type of input we want

Reply:
We can either remove this prop and handle the character count within `TextInput` directly (see other comments), or just pass this value if the types match - 

```ts
textLength={[TextInputType.Long, TextInputType.ExtraLong].includes(type) && TEXT_LIMIT[type]
```